### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by lacksfish**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ before_install:
  - cp appconfig.default.json appconfig.json
 
 script:
- - gulp
+ - ./node_modules/.bin/gulp
```
Comment (afk11) [inline .travis.yml]: maybe update readme. this change should be applied in the docker also

Comment (lacksfish) [reply .travis.yml]: Think it's running same gulp? Just being more specific.

Comment (annipier32) [reply .travis.yml]: What

**Comment by lacksfish** [comment]
Updating the dependencies is still a task worth undertaking, but as this PR aged I think new versions of dependencies have been published and a new dependency overhaul would be better of to start on the new master branch top commit. Closing this.

**Pull request closed without merging**